client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -209,6 +209,36 @@ const PluginMeta = React.createClass( {
 		}
 	},
 
+	maybeDisplayUnsupportedNotice() {
+		const { selectedSite, plugin } = this.props;
+
+		// Pressable prevents installation of some plugins, so we need to disable AT for them.
+		// More info here: https://kb.pressable.com/faq/does-pressable-restrict-any-plugins/
+		const unsupportedPlugins = [
+			'nginx-helper',
+			'w3-total-cache',
+			'wp-rocket',
+			'wp-super-cache',
+			'bwp-minify',
+		];
+
+		if ( selectedSite && ! selectedSite.jetpack && includes( unsupportedPlugins, plugin.slug ) ) {
```

Comment:
This catches business sites that are pre-transfer. But the incompatibility is specific to the hosting, not the transfer process, so we need to apply it to sites _after_ they transfer as well. Right now when testing this branch I can still install an incompatible plugin on a transferred site.